src/internal/nativeInterface.web.ts

```diff
@@ -0,0 +1,27 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @format
+ */
+
+import {NativeEventEmitter} from 'react-native';
```

Comment:
The use of "react-native" import in web contexts implies the need to use react-native-web.
In my opinion, this is too binding in addition to causing an increase in the bundle size

Reply:
I am curious on how should the NativeEventEmitter be imported, like creating it's definition instead of importing from `react-native`?